Update Jobs-Discord-Asked-Questions.md (#1)
mrfdev/Jobs: Resources/FAQ/Jobs-Discord-Asked-Questions.md

# Keep in mind that jobs.max.[amount] will bypass this setting
max-jobs: 3
```

> They also need [permission](https://Github.com/mrfdev/Jobs/blob/main/Resources/FAQ/Jobs-permissions.md) to join any job.

Reviewer: Imo this should keep the "note" prefix, as the information is an additional tip

### The plugin doesn't start!

> That sucks, but we require more info to help. Does the Jobs-related error message in your console say 'missing dependency, CMILib required, download it' ?

Reviewer: Maybe say about where to find the console MSG's first, before, to keep the flow

IE:
That sucks that it isn't working, but we need more info...
In your console there will be a msg relating to jobs...
But don't worry if you missed it... Latest.log...

If the error says...

First, please check that you're on the latest build of your service engine; type: `/ver` and see how far behind you are.

Update to the latest versions of [CMILib](https://www.spigotmc.org/resources/cmilib.87610/) and [Jobs](https://www.spigotmc.org/resources/jobs-reborn.4216/)

Try again. If it still fails, check the latest.log file to see the errors during startup and resolve those.

If you believe you've found a bug, gather the info about your server and go to Zrips's Github and submit a new issue for review.

Provide at least the following:
What the problem is, and how can it be reproduced
What you've done to mitigate it so far.
The output of the command /ver
Versions of other plugins related to your problem
include the entire error message.

---

### Jobs is lagging my server, shows in timings report.

That's not how the timings work. Jobs showing up as a lagging can mean many things. Another plugin can hang the main thread, forcing a queue to build up or other tasks to wait before completing.

We will need to know much more information, including your timings report.

> Going to [Github](https://github.com/mrfdev/Jobs/issues) and starting a new issue is the best way to deal with that.

Reviewer: This should be pointing to the zrips bug tracker, as this is only for the community documentation

---

### Where can I find a list of all Jobs Reborn placeholders?

Type: `jobs placeholders` to get the complete list in the console.

> You can also do this in-game with the command `/jobs placeholders`

Reviewer: This repeat information in an unneeded way: maybe remove it, and alter the original to just say to run the command anywhere with permission

---

### I asked a question, and I am getting ignored?

> To get the best help you can get, we need information, then Zrips on Github or the community members on Discord can help you better.

Reviewer: I know this wasn't introduced by you, but maybe
```suggestion
To get the best help possible, we need information, then Zrips on Github or the community members on Discord can help you better.
```

- Have some patience. Post the question, provide info, and just wait.

> - We always advise you to stay current and make frequent backups!

Reviewer: Probably not needed, as this advice is given to everyone, not specifically the reader

**# Economy manager:** `ver CMI`
**# Permission Manager:** `ver LuckPerms`

> - Keep an eye on the console or check the logs to make sure you spot any errors during the startup of any plugins. If you see an error, try to disable the plugin causing it and retry. If there's a need to paste extensive logs and error messages, use a Pastebin type service such as  you want to paste your big error msgs, please use a text storage site like [Hastebin](https://paste.md-5.net/)

Reviewer: The mention of errors appearing where someone took an action (like running a command / placing a block) was removed, and it should probably be put back

> - Keep an eye on the console or check the logs to make sure you spot any errors during the startup of any plugins. If you see an error, try to disable the plugin causing it and retry. If there's a need to paste extensive logs and error messages, use a Pastebin type service such as  you want to paste your big error msgs, please use a text storage site like [Hastebin](https://paste.md-5.net/)

Reviewer: The advice of disabling the plugin causing the error isn't great, as it may not be the root cause, and only a symptom.
Instead, advising people to read the root cause of the error might be helpful if it is solely user caused.
Tho if they are on latest, it should still be reported

```
And swap out `%item%` with the item you want it to be

> Each job has its own file, so you have to go through them and change it per file. 

Reviewer: ```suggestion
Each job has its own file, so you have to go through them and change it in each one. 
```

---

### How can I use (short) Jobs names (in chat, etc.)

My best bet is to check the `/jobs placeholders` and find the one you want to use and use that in your chat manager, holograms, etc. 

https://www.Zrips.net/jobs/chat-titles/

```
chat-display

> The display method of the title. Accepted values are full, title, job, shortfull, shorttitle, shortjob and none

Reviewer: Revert, to match the page it's quoting from

> Hi, is there any way to be able to open the jobs menu using /jobs and not /jobs browse

> Not by default, another plugin might be required to alias the /jobs command, ensuring that it opens /jobs browse instead of /jobs help. It would be nice to have a toggle. 

Reviewer: Suggestion, not introduced by you
```suggestion
Not by default, another plugin might be required to alias the `/jobs` command, ensuring that it opens `/jobs browse` instead of `/jobs help`. It would be nice to have a toggle. 
```
